dlt/pipeline/pipeline.py

```diff
@@ -920,7 +922,7 @@ def _get_destination_client(self, schema: Schema, initial_config: DestinationCli
                 "Dependencies for specific destinations are available as extras of dlt"
             )
 
-    def _get_staging_client(self, schema: Schema, initial_config: DestinationClientConfiguration = None) -> JobClientBase:
+    def ync(self, schema: Schema, initial_config: DestinationClientConfiguration = None) -> JobClientBase:
```

Comment:
do I see correctly? `ync1?

Reply:
this one should be deleted, I have done that now